### DISCUSSION TITLE
scorefile not formed as a seperate file but scores are shown in PDB file

### DISCUSSION THREAD

**Discussion by intomybioverse** [RosettaCommons/rosetta#86] [Rosetta]
Hello. I am working with the rosettaDNA application, and my score.sc file is not separately forming. The scores are rather shown in the designed PDB file (the output file). It is very difficult to sort the structures based on the energy score this way. My argument file (design.txt) and the PDB output file (design_output.txt) is attached here, please tell me how to get a separate score.sc file with all the desired values. 

[design.txt](https://github.com/user-attachments/files/16124976/design.txt)
[design_output.txt](https://github.com/user-attachments/files/16124974/design_output.txt)

**Comment by ajasja** [comment]
Not familiar with the rosettaDNA. You should be getting a separate score file on every rosetta run. But as a temp solution you can grep the scores from the PDBs using `grep "REMARK      total_score" *.pdb` or similar

**Comment by NooraAz** [comment]
I think after `-out:file:scorefile` you should specify the name of your score file. for example: `-out:file:scorefile myscorefile.sc`

**Reply by ajasja** [reply]
Nice catch:)

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
